.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
It looks like this is a useless file  （.vscode/settings.json）

Reply:
the file was lost in the commit is part of the configuration of a plugin for visual studio code, to help me with the translation